tailcfg/tailcfg.go

```diff
@@ -1313,6 +1332,20 @@ type PingResponse struct {
 }
 
 type MapResponse struct {
+	// MapSessionHandle optionally specifies a unique opaque handle for this
+	// stateful MapResponse session. Servers may not send it, and it's only sent
```

Comment:
Maybe "servers may choose not to send it", since I read this as "it is not permitted for a server to send this" at first?

Reply:
Done